mne/epochs.py

```diff
@@ -872,10 +877,22 @@ def _compute_mean_or_stderr(self, picks, mode='ave'):
 
         if self.preload:
             n_events = len(self.events)
-            fun = np.std if _do_std else np.mean
-            data = fun(self._data, axis=0)
+            modes = {"mean": lambda x: np.mean(x, axis=0),
+                     "std": lambda x: np.std(x, axis=0),
+                     "gfp": lambda x: np.sqrt((x * x).mean(axis=0))
+                     }
+            if mode in modes:
+                fun = modes[mode]
+            elif callable(mode):
+                pass
```

Comment:
fun = mode

this must not be tested...

Reply:
>this must not be tested...

That's why it reads WIP :|